src/Sylius/Bundle/OrderBundle/Form/Type/OrderItemType.php

```diff
@@ -45,9 +47,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('quantity', 'integer', [
                 'label' => 'sylius.form.order_item.quantity',
+                'constraints' => [
+                    new NotNull(),
+                    new Type(['type' => 'numeric']),
+                ],
             ])
             ->add('unitPrice', 'sylius_money', [
                 'label' => 'sylius.form.order_item.unit_price',
+                'constraints' => [
+                    new NotNull(),
+                    new Type(['type' => 'numeric']),
```

Comment:
variant can be validated, but unitPrice and quantity have defaults on models, which are used, so if you don't POST data on those fields, the defaults will be used and the validation passes. Any idea?